Update language of io_processing.md (#3935)
juicedata/juicefs: docs/en/introduction/io_processing.md

![](../images/internals-read.png)

Although prefetching works well for sequential reads, it might not be so effective for random reads on large files. It can cause read amplification and frequent cache eviction. Consider disabling prefetching using `--prefetch=0`. It is always hard to design cache strategy for random read scenarios. Two possible solutions are increasing the cache size to store all data locally or completely disabling the cache (`--cache-size=0`) and relying on a high-performance object storage service.

> Reading small files (smaller than the block size) is much easier. The entire file can be read in a single request. Since small files are cached locally during the write process, future reads are very fast due to the local caching. This approach provides impressive performance, as demonstrated by `juicefs bench`.

Reviewer: bench 命令是允许指定 block size 的，这一句的表述已经不再准确（也许从前 bench 命令只写小文件）

```suggestion
Reading small files (smaller than the block size) is much easier because the entire file can be read in a single request. Since small files are cached locally during the write process, future reads are fast.
```
